client/src/views/LevelUpView.tsx

```diff
@@ -74,8 +80,8 @@ const LevelUpView = () => {
     // Update brute data
     updateBrute((data) => (data ? ({ ...data, ...newBrute }) : null));
 
-    navigate(`/${brute.name}/cell`);
-  }, [Alert, brute, choices, navigate, updateBrute, updateData]);
+    navigate(xpNeededForNextLevel > brute.xp ? `/${brute.name}/cell` : `/${brute.name}/level-up`);
```

Comment:
Instead, just use `getXPNeeded(brute.level + 1)` directly here

Reply:
Sorry i'll have to watch some tutorial

Reply:
I think I did something wrong with git it pushed some files I never added